src/components/Modal/ModalWrapper/ModalWrapper.tsx

```diff
@@ -40,12 +40,15 @@ export const ModalWrapper = forwardRef(
           data-testid="modalOverlay"
           className="usa-modal-overlay"
           onClick={forceAction ? undefined : handleClose}
-          aria-controls={id}>
+          aria-controls={id}
+        >
           {children}
         </div>
       </div>
     )
   }
 )
 
+ModalWrapper.displayName = 'ModalWrapper'
```

Comment:
This one doesn't make sense to me. Why is it needed? Isn't the displayname already ModalWrapper? or am I misunderstanding what displayName is/does.